.github/workflows/build.yml

```
@@ -184,10 +194,27 @@
+      - uses: actions/checkout@v4
+
+      - name: Save Kubeconfig
+        run: |
+          doctl kubernetes cluster kubeconfig save --expiry-seconds 600 django-mindjunkies
+
+      - name: Run Django Migrations and Collectstatic Jobs
+        run: |
+          kubectl delete job django-migrate-job --ignore-not-found
+          kubectl apply -f k8s/jobs/django-migrate-job.yml
+          kubectl wait --for=condition=complete job/django-migrate-job --timeout=120s
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)